README.md

```diff
@@ -38,13 +38,20 @@ compile 'online.devliving:passcodeview:1.0.3'
 - `setText(CharSequence text)` - Set Passcode programmatically
 - `clearText()` - Clear Passcode
 - `getText()` - get entered Passcode
+- `swetError(boolean value)` - set the error state of view
```

Comment:
I guess that's a typo, should be `setError`?

Reply:
You're right, yes, thx, will update after my vacation in ~1 week